collectors/python.d.plugin/adaptec_raid/README.md

```diff
@@ -16,6 +7,10 @@ Executed commands:
 - `sudo -n arcconf GETCONFIG 1 LD`
 - `sudo -n arcconf GETCONFIG 1 PD`
 
+## Metrics
+
+You can find the list of metrics this module provides at [metrics.csv](https://github.com/netdata/netdata/blob/master/collectors/python.d.plugin/adaptec_raid/metrics.csv).
```

Comment:
> If you want, let's keep the wording in the README the same, so I can replace it with the table inside the documentation

Agree. But let's make the sentence shorter by removing some specific parts like "this module" - not all plugins/collectors have modules.

Can be just "See [metrics.csv](qq) for a list of metrics."